Fix issues in NFT tutorial - Closes #1059 (#1062)
LiskHQ/lisk-docs: modules/ROOT/pages/tutorials/nft.adoc

The NFT plugin shall provide an HTTP API that offers new endpoints for NFT related data from the blockchain.

Navigate out of the `nft_module` folder back into the `blockchain_app` folder.

If you haven't done it yet, add now an author to your `package.json` file.

> We will use this information in the plugin later on.

Reviewer: ```suggestion
This information will be used later in the plugin.
```
